test/e2e/specs/specs-playwright/wp-signup__screenshot.ts

```diff
@@ -0,0 +1,118 @@
+			let form = new FormData();
+			form.append( 'my_file', fs.createReadStream( 'login.png' ) );
+			await fetch( 'https://public-api.wordpress.com/wpcom/v2/screenshots', {
+				method: 'POST',
+				body: form,
+			} )
+			.then( response => response.json())
+			.then( response => expect( response?.upload_status ).toStrictEqual( 'success' ) );
+			await loginPage.signup();
+		} );
+
+		it( 'Set store cookie', async function () {
+			await BrowserManager.setStoreCookie( page, { currency: 'GBP' } );
+		} );
+
+		it( 'Screenshot signup page and sign up as new user', async function () {
+			const userSignupPage = new UserSignupPage( page );
+			await page.screenshot( { path: 'signup.png', fullPage: true } );
+			let form = new FormData();
+            form.append( 'my_file', fs.createReadStream( 'signup.png' ) );
+            await fetch( 'https://public-api.wordpress.com/wpcom/v2/screenshots', {
+            	method: 'POST',
+                body: form,
+            } )
+            .then( response => response.json())
+            .then( response => expect( response?.upload_status ).toStrictEqual( 'success' ) );
+			await userSignupPage.signup( email, username, signupPassword );
+		} );
+
+		it( 'Select a free .wordpress.com domain', async function () {
+			const domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( blogName );
+			await domainSearchComponent.selectDomain( '.wordpress.com' );
+		} );
+
+		it( 'Select WordPress.com Personal plan', async function () {
+			const signupPickPlanPage = new SignupPickPlanPage( page );
+			await signupPickPlanPage.selectPlan( 'Personal' );
+		} );
+
+		it( 'See secure payment', async function () {
+			cartCheckoutPage = new CartCheckoutPage( page );
+			await cartCheckoutPage.validateCartItem( 'WordPress.com Personal' );
+		} );
+
+		it( 'Enter billing and payment details', async function () {
+			const paymentDetails = DataHelper.getTestPaymentDetails();
+			await cartCheckoutPage.enterBillingDetails( paymentDetails );
+			await cartCheckoutPage.enterPaymentDetails( paymentDetails );
+		} );
+
+		it( 'Take screenshot of checkout page', async function () {
+			await page.screenshot( { path: 'checkout.png', fullPage: true } );
+			let form = new FormData();
+            form.append( 'my_file', fs.createReadStream( 'checkout.png' ) );
+            await fetch( 'https://public-api.wordpress.com/wpcom/v2/screenshots', {
+            	method: 'POST',
+                body: form,
+            } )
+            .then( response => response.json())
+            .then( response => expect( response?.upload_status ).toStrictEqual( 'success' ) );
+			const navbarComponent = new NavbarComponent( page );
+			await navbarComponent.clickCloseCheckout();
+		} );
+	} );
+
+	describe( 'Delete user account', function () {
+		it( 'Close account', async function () {
+			const closeAccountFlow = new CloseAccountFlow( page );
+			await closeAccountFlow.closeAccount();
+		} );
```

Comment:
This is more of a question - does what you are trying to accomplish require the use of a new account created at the time of the script execution?

Reply:
To expand on my previous comment, is it possible to achieve the screenshot of Checkout TOS by simply logging into an existing user, add an item to the cart, navigate to the checkout page and take screenshots?

Where possible, we try to avoid creating new users or sites to help reduce burden on production databases.

Reply:
Thanks for the feedback, I've removed the creation of new user.